addons/website_slides/models/slide_channel.py

```diff
@@ -143,6 +151,44 @@ def _compute_website_url(self):
     def change_visibility(self):
         pass
 
+    @api.multi
+    def action_redirect_to_members(self):
+        view_list_id = self.env.ref('website_slides.view_slide_channel_partner_tree').id
```

Comment:
I think we could have at least one action in data, notably if we want to add a menu entry in debug mode somewhere to have access to channel members. Then just read it and tweak domain / context if necessary to avoid having to declare an action in python.

Reply:
Should be ok now if I understood what you meant correctly.
(Could be wise to double check though).